www.chuyu.me/es.xml

```diff
@@ -1,6 +1,6 @@
 <Strings>
   <String Src="%lang%" Dst="es"/>
-  <String Src="%description%" Dst="Integrar actualizaciones y controladores, etc, limpiar WinSxS y el caché de paquetes etc, personalizar la imagen del SO, aumentar el su proceso de implementación de Windows."/>
+  <String Src="%description%" Dst="Integrar actualizaciones y controladores, etc, limpieza de WinSxS y el caché de paquetes etc, personalizar la imagen del sistema operativo, aumentar el su proceso de implementación de Windows."/>

```

Comment:
The correct thing would be to display everything without cutting on the page, enter to display all the complete words.